ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts

```diff
@@ -35,6 +35,8 @@ const getRenderActivityKeyByActivityCode = (activityCode: ActivityCode): RenderA
       return RenderActivityKey.MedicalRecordUpload;
     case ActivityCode.PatientSurvey:
       return RenderActivityKey.PatientSurvey;
+    case ActivityCode.ChildContact:
+      return RenderActivityKey.About;
```

Comment:
Returning About as a Render-Activity-Code

Reply:
I believe you have caught the bug root cause.

**One note**:  suggesting move that check to line 27 where the condition has already been returning `RenderActivityKey.About`

Reply:
sure, Makes sense.